packages/airnode-node/src/handlers/process-http-signed-data-request.ts

```diff
@@ -25,7 +25,11 @@ export async function processHttpSignedDataRequest(
 
   const requestId = randomHexString(16);
   const logOptions = buildBaseOptions(config, { requestId });
-  const airnodeAddress = wallet.getAirnodeWallet(config).address;
+  const airnodeWalletPrivateKey = getEnvValue('AIRNODE_WALLET_PRIVATE_KEY');
+  if (!airnodeWalletPrivateKey) {
+    return [new Error('Missing Airnode wallet private key in environment variables.'), null];
+  }
+  const airnodeAddress = wallet.getAirnodeWalletWithPrivateKey(airnodeWalletPrivateKey).address;
```

Comment:
I don't like much that we're doing these checks at so many places but can't really see a better solution :confused: